.bazelversion

```diff
@@ -1 +1 @@
-3.5.0
+4.0.0
```

Comment:
https://github.com/bazelbuild/continuous-integration/issues/1060

Reply:
Yeah, I added that to be able to test locally since the flags are 4.0 only, but it actually seemed to work ok in the buildkite job... 

![image](https://user-images.githubusercontent.com/65615298/117724216-1384ed00-b1a9-11eb-8096-79456a6bbd16.png)

Reply:
Do you need to build and push a new image when I change that Dockerfile? I added what the job recommended but it looks like it's still the same image.
```
Bazel binaries for all official releases can be downloaded from here:
  https://github.com/bazelbuild/bazel/releases
You can download the required version directly using this command:
  (cd "/usr/local/lib/bazel/bin" && curl -fLO https://releases.bazel.build/4.0.0/release/bazel-4.0.0-linux-x86_64 && chmod +x bazel-4.0.0-linux-x86_64)
  ```